Packs/ExportToXLSX/Scripts/ExportToXLSX/ExportToXLSX.py

```diff
@@ -62,40 +64,36 @@ def main():
         data = args.get("data")
         file_name = args.get("file_name")
         sheet_name = args.get("sheet_name")
-        headers = args.get("headers", None)
-        is_bold = argToBoolean(args.get("bold", 'true'))
-        is_border = argToBoolean(args.get("border", 'true'))
+        headers = args.get("headers")
+        is_bold = argToBoolean(args.get("bold", "true"))
+        is_border = argToBoolean(args.get("border", "true"))
 
-        sheets = sheet_name.split(",")
+        sheets = argToList(sheet_name)
         data = parse_data(data, sheets)
 
-        if len(sheets) != len(data):
-            raise ValueError("Number of sheet names should be equal to the number of data items.")
-
         if headers:
-            headers_list = headers.split(";")
+            headers_list = argToList(headers, separator=";")
 
             if len(sheets) != len(headers_list):
                 raise ValueError("Number of sheet headers should be equal to the number of sheet names")
         else:
             headers_list = None
 
-        workbook = xlsxwriter.Workbook(file_name)
+        with xlsxwriter.Workbook(file_name) as workbook:
```

Comment:
you now have it as an import
```suggestion
        with Workbook(file_name) as workbook:
```